src/txn/txn_ckpt.c

```diff
@@ -1194,15 +1140,14 @@ __txn_checkpoint(WT_SESSION_IMPL *session, const char *cfg[])
      */
     __checkpoint_wait_reduce_dirty_cache(session);
 
-    /* Tell logging that we are about to start a database checkpoint. */
-    if (full && logging)
-        WT_ERR(__wt_txn_checkpoint_log(session, full, WT_TXN_LOG_CKPT_PREPARE, NULL));
+    /* Tell logging that we are about to start a full database checkpoint. */
+    if (logging)
+        WT_ERR(__wt_txn_checkpoint_log(session, true, WT_TXN_LOG_CKPT_PREPARE, NULL));
```

Comment:
What is the meaning of the second argument to `__wt_txn_checkpoint_log`? If it's no longer possible for it to be `false`, I think we should consider dropping it. Likewise for `__wt_meta_sysinfo_set`.

Reply:
It can still be false in calls to __wt_txn_checkpoint_log().

* __wt_txn_checkpoint_log() with true: __wti_connection_close(), __txn_checkpoint() thrice,
* __wt_txn_checkpoint_log() with false: __wt_meta_track_off(), __checkpoint_tree() twice

Both __wt_meta_sysinfo_set() and __wt_meta_sysinfo_update() are only called with full = true. Removed that parameter from both of them.

__wt_meta_sysinfo_remove() is still called with full = false by __wt_meta_sysinfo_clear().

Done

Reply:
Thanks!